### PR TITLE
Removal of subgraph & apollo

### PR DESCRIPTION
**Pull request by joehawary**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,6 @@
   "version": "1.3.0-beta.3",
   "private": true,
   "dependencies": {
-    "@apollo/client": "^3.3.14",
```
Comment (nuurcodes) [inline package.json:6]: :)

Comment (joehawary) [reply package.json:6]: 👋  apollo has left the building

File: src/helpers/formatTokenSymbol.ts [hunk 2]
```diff
@@ -1,22 +1,42 @@
...
 export const formatTokenSymbol = async (
   address: string,
   tokenSymbol: string,
-  chainId: number
+  chainId: number,
+  signer: Signer
 ) => {
   const isLp = tokenSymbol === 'UNI-V2';
 
-  if (chainId !== 1 || !isLp) {
```
Comment (nuurcodes) [inline src/helpers/formatTokenSymbol.ts:11]: Glad we are showing Rinkeby and Gorli some love

Comment (joehawary) [reply src/helpers/formatTokenSymbol.ts:11]: Lets us also see how our code performs with 100 subscriptions 😆

File: src/helpers/formatTokenSymbol.ts [hunk 3]
```diff
@@ -1,22 +1,42 @@
...
+  if (!isLp) {
     // No Uniswap graph for Rinkeby
     return tokenSymbol;
   }
   try {
-    const { data } = await apolloClient.query({
-      query: GET_PAIR_DATA(address.toLocaleLowerCase()),
-    });
-    return data.pair.token0.symbol + ' - ' + data.pair.token1.symbol + ' LP';
+    const lpContract = new ethers.Contract(address, IUniswapV2Pair.abi, signer);
```
Comment (nuurcodes) [inline src/helpers/formatTokenSymbol.ts:18]: Love this 🥇

**Review by nuurcodes — APPROVED** [review]
Great work

**Pull request merged**